cobbler/action_buildiso.py

```diff
@@ -397,7 +397,10 @@ def generate_netboot_iso(self,imagesdir,isolinuxdir,profiles=None,systems=None,e
                    else:
                       append_line += " dns=%s" % my_dns
                 if dist.breed in ["ubuntu","debian"]:
-                   append_line += " netcfg/get_nameservers=%s" % ",".join(my_dns)
+                   if type(my_dns) == list:
```

Comment:
I believe the following suggestion is the recommended way to do type checks in Python. Your check will only work with an _exact_ list. The proposed change would also work with an object which is a subclass of a `list`.

```suggestion
                   if isinstance(my_dns, list):
```

Reply:
@SchoolGuy I certainly agree, with this comment but, if you check the code, I used the same style which was already used in the script. If we change this one we need to change the other occurrences.

Reply:
Ah okay. Then let's not change the pattern.